qt/src/hocr/HOCRPdfExporter.cc

```diff
@@ -784,5 +797,34 @@ void HOCRPdfExporter::addInfoClicked() {
 		ui.lineEditTitle->setText(source->title);
 		ui.lineEditSubject->setText(source->subject);
 		ui.lineEditProducer->setText(source->producer);
+
+		PoDoFo::EPdfVersion sourcePdfVersion;
+		switch(source->pdfVersionMinor) {
+		case 0:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_0;
+			break;
+		case 1:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_1;
+			break;
+		case 2:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_2;
+			break;
+		case 3:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_3;
+			break;
+		case 4:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_4;
+			break;
+		case 5:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_5;
+			break;
+		case 6:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_6;
+			break;
+		default:
+			sourcePdfVersion = PoDoFo::EPdfVersion::ePdfVersion_1_7;
+			break;
+		}
```

Comment:
What about `static_cast<PoDoFo::EPdfVersion>(PoDoFo::EPdfVersion::ePdfVersion_1_0 + source->pdfVersionMinor)`?

Reply:
Yes, we can... in this case we will rely on internal representation of this enum. But ok

Reply:
The enum is part of the public API, so it is never supposed to change.

Reply:
You might want to add a fallback to 1.7 of major is >= 2 for future proofness also.

Reply:
And clearly this fallback is updated as PoDoFo picks up possible new versions, if we remember to do so. But if we forget to do so, at least it does not end up using version 1.0 or another ancient version.